brage-import/src/main/java/no/sikt/nva/brage/migration/merger/CristinImportPublicationMerger.java

```diff
@@ -166,33 +172,61 @@ private Set<AdditionalIdentifier> mergeAdditionalIdentifiers() {
         return additionalIdentifiers;
     }
 
-    private Publication fillNewPublicationWithMetadataFromBrage(Publication publicationForUpdating) {
-        publicationForUpdating.getEntityDescription().setDescription(getCorrectDescription());
-        publicationForUpdating.getEntityDescription().setAbstract(getCorrectAbstract());
-        publicationForUpdating.setAssociatedArtifacts(determineAssociatedArtifacts());
-        return publicationForUpdating;
-    }
-
     private AssociatedArtifactList determineAssociatedArtifacts() {
         if (cristinPublication.getAssociatedArtifacts().isEmpty()) {
             return bragePublicationRepresentation.publication().getAssociatedArtifacts();
         }
-        if (!hasAdministrativeAgreement(cristinPublication) && hasAdministrativeAgreement(bragePublicationRepresentation.publication())) {
-            var administrativeAgreements = extractAdministrativeAgreements(bragePublicationRepresentation.publication());
+        if (!hasAdministrativeAgreement(cristinPublication) && hasAdministrativeAgreement(
+            bragePublicationRepresentation.publication())) {
+            var administrativeAgreements = extractAdministrativeAgreements(
+                bragePublicationRepresentation.publication());
             cristinPublication.getAssociatedArtifacts().addAll(administrativeAgreements);
             return cristinPublication.getAssociatedArtifacts();
         }
-        if (shouldUseBrageArtifacts()) {
+        if (shouldOverWriteWithBrageArtifacts()) {
```

Comment:
Not sure myself about this.
Do we want to replace everything when one of the files are better than the one present?